docs/how-to-build-with-ddn/with-openapi.mdx

```diff
@@ -0,0 +1,261 @@
+---
+sidebar_position: 7
+sidebar_label: With OpenAPI
+description: "Learn the basics of Hasura DDN and how to get started with HTTP APIs using OpenAPI."
+keywords:
+  - hasura ddn
+  - graphql api
+  - getting started
+  - guide
+  - openapi
+  - open api
+  - http api
+  - rest api
+  - restful api
+  - custom api
+---
+
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# Get Started with Hasura DDN and OpenAPI
+
+## Overview
+
+This tutorial takes about fifteen minutes to complete. You'll learn how to:
+
+- Set up a new Hasura DDN project
+- Connect it to HTTP APIs defined in an OpenAPI Document
+- Generate Hasura metadata
+- Create a build
+- Run your first query
+
+Additionally, we'll familiarize you with the steps and workflows necessary to iterate on your API.
+
+This tutorial assumes you're starting from scratch. We will use the [Petstore API](https://petstore3.swagger.io/) in
+this tutorial, but you can use any OpenAPI document.
+
+<Prereqs />
+
+## Tutorial
+
+### Step 1. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your OpenAPI connector
+
+```sh title="In your project directory, run:"
+ddn connector init my_openapi -i
+```
+
+From the dropdown, start typing `openapi` and hit enter to accept the default port. Then, provide the following values:
```

Comment:
Connector doesn't ask for port. Also, can we just have a line about the values for each of the NDC_OAS_LAMBDA_PRETTY_LOGS and NDC_OAS_FILE_OVERWRITE variables and what they do. Eg: 

The NDC_OAS_LAMBDA_PRETTY_LOGS environment variable shows more human readable logs in the output. 
The NDC_OAS_FILE_OVERWRITE environment variable tells the OpenAPI connector to overwrite existing generated files when re-introspecting your OpenAPI Document.

Reply:
Resolved in [0c04764](https://github.com/hasura/ddn-docs/pull/928/commits/0c047644947ee855bc56ee3162defb83c8d673e4)